string[] object_list
---
string status